test/a11y-wand-test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Tests for a11yTextWand
+ */
+
+let assert = require("assert");
+let mockDom = require("./mock-dom");
+
+describe("Screen Reader Wand", function() {
+    let plugin = null;
+
+    before(function(done) {
+        mockDom.createDom(function() {
+            let ScreenReaderWand = require("../plugins/a11y-text-wand");
+            plugin = new ScreenReaderWand();
+
+            document.body.innerHTML = `
+                <article id="article">
+                    <p>This paragraph is visible.</p>
+                    <p id="p-with-hidden">This paragraph has a <span id="span" aria-hidden="true">hidden</span> span.</p>
+                    <p id="hidden-p" aria-hidden="true">This paragraph is hidden.</p>
+                </article>
+            `;
+
+            done();
+        });
+    });
+
+    it("should handle aria-hidden correctly", function() {
```

Comment:
What do you think about expressing what "correctly" means here?

Perhaps "should ignore elements marked with aria-hidden" or similar? That way if the test fails, it might be clearer why.

Reply:
👍